Vagrantfile

```diff
@@ -3,19 +3,24 @@
 
 Vagrant.require_version '>= 1.4.1'
 
-%w(vagrant-omnibus).each do |plugin|
+%w(vagrant-ohai vagrant-omnibus).each do |plugin|
   fail "Missing #{plugin}. Please install it!" unless Vagrant.has_plugin? plugin
 end
 
 @network = {
   chef:         { ip: '33.33.33.33', hostname: 'chef', ports: { 4000 => 4000 } },
-  c1_search:    { ip: '33.33.33.10', hostname: 'search.splunk', ports: { 8001 => 8000, 8091 => 8089 } },
   c1_master:    { ip: '33.33.33.11', hostname: 'master.splunk', ports: { 8002 => 8000, 8092 => 8089 } },
   c1_slave1:    { ip: '33.33.33.12', hostname: 'slave01.splunk', ports: { 8003 => 8000, 8093 => 8089 } },
   c1_slave2:    { ip: '33.33.33.13', hostname: 'slave02.splunk', ports: { 8004 => 8000, 8094 => 8089 } },
   c1_slave3:    { ip: '33.33.33.14', hostname: 'slave03.splunk', ports: { 8005 => 8000, 8095 => 8089 } },
-  s_standalone: { ip: '33.33.33.20', hostname: 'splunk2', ports: { 8006 => 8000, 8096 => 8089 } },
-  s_license:    { ip: '33.33.33.30', hostname: 'splunk-license', ports: { 8007 => 8000, 8097 => 8089 } },
+  c1_search1:   { ip: '33.33.33.15', hostname: 'search01.splunk', ports: { 8006 => 8000, 8096 => 8089 } },
+  c1_search2:   { ip: '33.33.33.16', hostname: 'search02.splunk', ports: { 8007 => 8000, 8097 => 8089 } },
+  c1_search3:   { ip: '33.33.33.17', hostname: 'search03.splunk', ports: { 8008 => 8000, 8098 => 8089 } },
+  c1_search4:   { ip: '33.33.33.18', hostname: 'search04.splunk', ports: { 8009 => 8000, 8099 => 8089 } },
+  c1_deployer:  { ip: '33.33.33.28', hostname: 'deployer.splunk', ports: { 8010 => 8000, 8100 => 8089 } },
+  s_standalone: { ip: '33.33.33.20', hostname: 'splunk2', ports: { 8011 => 8000, 8101 => 8089 } },
+  s_license:    { ip: '33.33.33.30', hostname: 'splunk-license', ports: { 8012 => 8000, 8102 => 8089 } },
+  s_search:    { ip: '33.33.33.31', hostname: 'splunk-search', ports: { 8013 => 8000, 8103 => 8089 } },
```

Comment:
minor, why change it to s_search and ports 8013,8103?  Since it's part of the (indexer) cluster I think we should keep the c1 prefix but I'm not sure what would make the most sense for the latter part.  Maybe c1_search_standalone? We could also leave it on 8001,8091.